pkg/kubelet/cm/container_manager_windows.go

```diff
@@ -111,16 +111,12 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	cm.topologyManager = topologymanager.NewFakeManager()
 
-	klog.InfoS("Creating device plugin manager", "devicePluginEnabled", devicePluginEnabled)
-	if devicePluginEnabled {
-		cm.deviceManager, err = devicemanager.NewManagerImpl(nil, cm.topologyManager)
-		cm.topologyManager.AddHintProvider(cm.deviceManager)
-	} else {
-		cm.deviceManager, err = devicemanager.NewManagerStub()
```

Comment:
Is ManagerStup used anywhere else? If not, should we remove it as well?

Reply:
Thanks, doesn't look like it is being used anywhere else. Going to remove it.

Reply:
Its used in the e2e test

Reply:
git grep doesn't think so :)
```
ed@devel kubernetes (master) $ git grep NewManagerStub
pkg/kubelet/cm/container_manager_linux.go:              cm.deviceManager, err = devicemanager.NewManagerStub()
pkg/kubelet/cm/container_manager_windows.go:            cm.deviceManager, err = devicemanager.NewManagerStub()
pkg/kubelet/cm/devicemanager/manager_stub.go:// NewManagerStub creates a ManagerStub.
pkg/kubelet/cm/devicemanager/manager_stub.go:func NewManagerStub() (*ManagerStub, error) {
```

Reply:
Yeah, I checked and we don't need the `NewManagerStub` for node e2e tests (in both [test/e2e_node/device_plugin_test.go ](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/device_plugin_test.go) and [test/e2e_node/device_manager_test.go](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/device_manager_test.go) and  ). My understanding is that for node e2e tests we run an instance of Kubelet and use device manager provided by Kubelet for testing and hence don't need a stub. Would you prefer if we left it in in case we need it in the future?

Reply:
This is what I was thinking of, which is different:
https://github.com/kubernetes/kubernetes/blob/master/test/images/sample-device-plugin/sampledeviceplugin.go#L89